### DISCUSSION TITLE
mssql数据库连接失败

### DISCUSSION THREAD

**Discussion by rebron1900** [prisma/prisma#10857] [Q&A] [answered]
执行下面这条命令时报错
google translate: Execute the following command times wrong
```
npx prisma migrate dev --name init 
```
错误信息：
google translate: error message
```
P1011: Error opening a TLS connection: The TLS settings didn't allow the connection to be established. Please review your connection string.
```
---
我后来在issue上看到有人说添加`TrustServerCertificate=true`可以解决问题，我添加后提示错误信息如下：
google translate: I see some people say that on the issue later add ` TrustServerCertificate = true ` can solve the problem, after I add a prompt error message is as follows:
```
Error in migration engine.
Reason: [libs\sql-schema-describer\src\mssql.rs:315:30] called `Result::unwrap()` on an `Err` value: "Couldn't parse default value: `create default d_password as 'D,73' `"
```

**Comment by janpio** [comment, marked as answer]
That looks like a bug. Can you open a bug issue please and include all the information the template asks for? Thanks.

**Reply by rebron1900** [reply]
我找了一些资料，发现我使用这个命令 `node --tls-min-v1.0 app.js` 启动时可以正常连接数据库 
请问这是我数据库的版本太低了么？不支持最新的TLS协议么？

_I did some research and found that I used this command node -- tls-min-v1.0 app. JS booting allows you to connect to the database normally_
_Is this version of my database too low? Doesn’t support the latest TLS protocol?_

>Microsoft SQL Server Management Studio						10.50.6000.34
Microsoft Analysis Services 客户端工具						10.50.6000.34
Microsoft 数据访问组件 (MDAC)						10.0.14393.0
Microsoft MSXML						3.0 6.0 
Microsoft Internet Explorer						9.11.14393.0
Microsoft .NET Framework						2.0.50727.8813
操作系统						6.3.14393

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
